### PR TITLE
AT_006.04 | Entity > Reference values > Delete the record

### PR DESCRIPTION
**Pull request by HannaRealKrasik**
https://trello.com/c/UWrDjonK/856-at00604-entity-reference-values-delete-the-record

add testDelete

### CODE REVIEW

**Review by viktoriiamishchenko — APPROVED** [review]

File: src/test/java/EntityReferenceValues1Test.java [hunk 1]
```diff
@@ -9,6 +9,10 @@
 
 
 public class EntityReferenceValues1Test extends BaseTest {
+     final static String TDS_WITH_VALUES = "//tr[@data-index='0']/td[@class]";
+     final static String CARD_BODY = "//div[@class='card-body ']";
+     final static String BIN = "//a[@href='index.php?action=recycle_bin']";
```
Comment (Stan575) [inline src/test/java/EntityReferenceValues1Test.java:14]: would be reasonable to make variables of type By

**Review by Stan575 — COMMENTED** [review]
test case says: The record is deleted, disappears from the existing list and is present in the “Recycle bin”.
Asking just with clarification purpose, what is criteria for a record to be present in recycle bin?

File: src/test/java/EntityReferenceValues1Test.java (hunk 1 above)
Comment (Stan575) [reply src/test/java/EntityReferenceValues1Test.java:14]: @HannaRealKrasik do my comments make sense for you?

Comment (HannaRealKrasik) [reply src/test/java/EntityReferenceValues1Test.java:14]: if I got write, here is Assert checks that record in bin (number 1 shows it) Assert.assertEquals(getDriver().findElement(By.xpath(BIN)).getText(), "delete_outline\n1" );
this is how looks empty bin:
Assert.assertEquals(getDriver().findElement(By.xpath(BIN)).getText(), "delete_outline" );

or I got wrong your question?

Comment (HannaRealKrasik) [reply src/test/java/EntityReferenceValues1Test.java]: update constants, now it of type By

**Review by Stan575 — APPROVED** [review]

**Pull request merged**